### PR TITLE
virttest/utils_test.qemu: Add func() for setup runlevel in guest.

### PR DESCRIPTION
**Pull request by CongSmile**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1261771

Previous PR under virt-test: autotest/virt-test#2310

### CODE REVIEW

**Comment by CongSmile** [comment]
Hi @xutian 

The previous PR has been closed under virt-test which you have already acked,
could you help review the new PR under avocado.

Thanks.

**Comment by PandaWei** [comment]
In my opinion,
The prefix 'avocado-vt' in commit message is not necessary.
and it makes the title line verbose.

File: virttest/utils_test/qemu.py [hunk 1]
```diff
@@ -146,6 +147,32 @@ def clear_win_driver_verifier(session, vm, timeout=300):
...
+    :param session: VM session.
+    """
+    expect_runlevel = params.get("expect_runlevel", "3")
+    cmd = "runlevel"
+
+    if utils_misc.wait_for(lambda: session.cmd_status(cmd) == 0, 15):
+        ori_runlevel = session.cmd(cmd)
+    else:
+        ori_runlevel = "0" 
+
```
Comment (PandaWei) [inline virttest/utils_test/qemu.py]: The following is a simplicity way, Just suggestion
ori_runlevel = "0"
if utils_misc.wait_for(lambda: session.cmd_status(cmd) == 0, 15):
    ori_runlevel = session.cmd(cmd)

**Comment by CongSmile** [comment]
Hi @PandaWei 

Thanks for your comment.

I have updated the code, could you help review it again?

Thanks.

**Comment by PandaWei** [comment]
LGTM

**Comment by CongSmile** [comment]
Hi @ldoktor 

I have updated the code according to your comments,
could you help review it again?

Thanks.

**Comment by ldoktor** [comment]
Thanks for the modification, apart from unused import it looks good.

**Comment by CongSmile** [comment]
Thanks @ldoktor  :+1: 

Remove the import now.

**Pull request merged**